dApp/client/src/pages/casino/Roulette.tsx

```diff
@@ -1,180 +1,282 @@
+            </div>
+          </div>
+        </div>
+      </div>
+      <div className='flex flex-col gap-4 items-center justify-center'>
+        {win ? (
+          <div className='alert alert-success shadow-lg m-5'>
+            <div>
+              <svg
+                xmlns='http://www.w3.org/2000/svg'
+                className='stroke-current flex-shrink-0 h-6 w-6'
+                fill='none'
+                viewBox='0 0 24 24'>
+                <path
+                  strokeLinecap='round'
+                  strokeLinejoin='round'
+                  strokeWidth='2'
+                  d='M9 12l2 2 4-4m6 2a9 9 0 11-18 0 9 9 0 0118 0z'
+                />
+              </svg>
+              <span>We have a winner!</span>
+            </div>
+          </div>
+        ) : (
+          <></>
+        )}
+
+        <div className='card flex flex-row'>
+          <div className={'stats stats-vertical shadow rounded-2xl border-4 border-primary'}>
+            <div className='stat'>
+              <div className={'stat-title text-primary'}>Winning Number</div>
+              <div className='stat-value text-primary pb-2'>
+                {gameStats ? gameStats.winningNumber : '__'}
+              </div>
+              <div className='stat-desc'>
+                {gameStats ? `Parity: ${gameStats.winningParity.toUpperCase()}` : 'Spin to Play'}
+              </div>
+              <div className='stat-desc'>
+                {gameStats ? `Color: ${gameStats.winningColor.toUpperCase()}` : ''}
+              </div>
+            </div>
+            <div className='stat'>
+              <input
+                type='text'
+                placeholder={autoChoose ? chosenPocket : 'Enter winning number'}
+                className={'input input-bordered input-primary w-full max-w-xs ' + textColor()}
+                disabled={autoChoose}
+                onChange={({ target: { value } }) => {
+                  if (0 <= parseInt(value) && parseInt(value) <= 36) {
+                    choosePocket(value);
+                  } else {
+                    choosePocket(undefined);
+                  }
+                }}
+              />
+              <div className='divider'>OR</div>
+              <div className='form-control'>
+                <label className='label cursor-pointer'>
+                  <span className='label-text text-secondary'>I'm feeling lucky!</span>
+                  <input
+                    type='checkbox'
+                    className='checkbox'
+                    checked={autoChoose}
+                    onChange={() => {
+                      choosePocket(undefined);
+                      handleChangePocket();
+                    }}
+                  />
+                </label>
+              </div>
+            </div>
+
+            <div className='stat'>
+              <div className='stat-title'>Bet Secured</div>
+              <div className='stat-value'>{betAmount} DEX</div>
```

Comment:
This still says DEX? Which I put as a placeholder. Should it say the symbol of our token?